fabric/src/main/java/ca/fxco/memoryleakfix/fabric/MemoryLeakFixFabric.java

```diff
@@ -16,4 +22,76 @@ public class MemoryLeakFixFabric implements ModInitializer {
     public void onInitialize() {
         MemoryLeakFix.init();
     }
+
+    public static void forceLoadAllMixinsAndClearSpongePoweredCache() {
+        String[] fabricLoaderVersion = FabricLoaderImpl.VERSION.split("\\.");
+        if (fabricLoaderVersion.length != 3) {
+            return;
+        }
+        int major = Integer.parseInt(fabricLoaderVersion[0]);
+        if (major != 0) {
+            return;
+        }
+        int minor = Integer.parseInt(fabricLoaderVersion[1]);
+        boolean runLeakFix = false;
+        if (minor < 14) {
+            runLeakFix = true;
+        } else if (minor == 14) {
+            int patch = Integer.parseInt(fabricLoaderVersion[2]);
+            if (patch < 15) {
+                runLeakFix = true;
+            }
+        }
+        if (runLeakFix) { // Must be fabric loader version smaller than v0.14.14. Patched in v0.14.15

```

Comment:
what if we just use Fabrics version comparing for this?
`if (Version.parse(FabricLoaderImpl.VERSION).compareTo(Version.parse("0.14.15")) < 0) {`

Reply:
hehe ya works for me